tests/functional/s/singledispatch_method.py

```diff
@@ -0,0 +1,41 @@
+"""Tests for singledispatch-method"""
+# pylint: disable=missing-class-docstring, missing-function-docstring,too-few-public-methods
+# pylint: disable=no-name-in-module
```

Comment:
Which module is reporting this?

Reply:
Oh, just realized this only happens for py37 cause `tests/functional/s/singledispatch_method.py` still imports `singledispatchmethod` for testing the right combination of `singledispatchmethod` and `classmethod`. I'll create a py37 and a py38+ versions of that test so I don't do that for py37.